File: Leetcode/1672/Linear_Search_Algo/main.cs
public class Solution {
    public int MaximumWealth(int[][] accounts) {
        int max = 0;
        foreach (int[] customer in accounts) {
            int transactions = 0;
            foreach (int transaction in customer) {
                transactions += transaction;
            }
            if (max < transactions) {
                max = transactions;
            }
        }
        return max;
    }
}

File: Leetcode/1672/Linear_Search_Algo/main.kt
class Solution {
    fun maximumWealth(accounts: Array<IntArray>): Int {
        var max: Int = 0
        for (customer in accounts) {
            var transactions: Int = 0
            for (transaction in customer) {
                transactions += transaction
            }
            if (max < transactions) {
                max = transactions
            }
        }
        return max   
    }
}